channeldb/channel.go

```diff
@@ -1680,7 +1680,7 @@ func serializeCommitDiff(w io.Writer, diff *CommitDiff) error {
 		return err
 	}
 
-	if err := diff.CommitSig.Encode(w, 0); err != nil {
+	if err := WriteElements(w, diff.CommitSig); err != nil {
```

Comment:
be coupled with the last commit? o/w running just this commit could break your db

Reply:
Good idea, done.